src/apis/notion.ts

```
@@ -1,13 +1,11 @@
-  //   pageId,
-  // );
-  
-  const recordMap = axios.get("https://notion-api.splitbee.io/v1/page/" + pageId ).then( res => res.data );
-
-  return recordMap;
+export const getNotionPage = async (pageId: string):Promise<ExtendedRecordMap> => {
+  try {  
+    const res = await groupsApi.get(`/notion/${pageId}`);
+    return res.data;  
+  } catch (error) {
+    throw error;  
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)